apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -34,47 +35,60 @@ const checkboxStoryMeta: CheckboxStoryMeta = {
             description: 'The value of the checkbox (used as a key/value pair in HTML forms with `name`).',
             control: 'text',
         },
+
         name: {
             description: 'The name of the checkbox (used as a key/value pair with `value`). This is required in order to work properly with forms.',
             control: 'text',
             defaultValue: {
                 summary: defaultArgs.name,
             },
         },
+
         label: {
             description: 'The visible label for the checkbox',
             control: 'text',
             defaultValue: {
                 summary: defaultArgs.label,
             },
         },
+
         checked: {
-            description: 'Indicates whether or not the checkbox is checked.',
+            description: 'The checked state of the checkbox.',
             control: 'boolean',
             defaultValue: {
                 summary: defaultArgs.checked,
             },
         },
+
+        defaultChecked: {
+            description: 'The default checked state of the checkbox.',
```

Comment:
Is it worth being totally explicit, so that:

```suggestion
            description: 'The default checked state of the checkbox (does not reflect if the checkbox is currently checked).',
```

Reply:
Updated to be _even more_ explicit 😄